README.md

```diff
@@ -1 +1,83 @@
-# Capstone2
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# JavaScript Capstone - API-based webapp
+
+> The JavaScript capstone project is build on [MealDB](www.themealdb.com/api/json/v1/1/filter.php?c=Seafood), an external API, and an [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) which is used to record the user interactions including likes, and comments.
+
+-     The main page shows a list or a collection of meals (retrieved from an ­external API) that you can like.
+-     A popup window with more data about an item that you can use to comment on the different meals and also see a list of comments.
+-     In the pop-up window you can see the main ingredients, needed for each recipe, a procedure  to follow for the preparation of each meal.
+-     The popup window consists of two buttons, one that connects to a YouTube video explaining  step-by-step of each recipe, and another one that links to official website.
+-     The application shows the number of likes and comments for each meals.
+  ­
+
+## Built With
+
+- HTML & CSS
+- Bootstrap
+- JavaScript(ES6)
+- Webpack
+- [HTML, CSS & JavaScript Linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js)
+- Jest Testing Library
+
+## Live Demo
+
+Live at [MealDB](https://kanzatahreem.github.io/JavaScript-Capstone/)
+
```

Comment:
- [ ] As required for the project, kindly make sure the maximum length of the video is **5 minutes**, right now, it exceeded the required length.
**I think it can be good if you add its link in your README file to make it accessible for users(You can create a new section in your README(e.g. after the `## Live Demo` section) and add the link there)**. I know you can do that :muscle: :muscle: